gridstatus/ercot.py

```diff
@@ -2142,6 +2145,32 @@ def _handle_price_corrections(self, docs, verbose=False):
 
         return df
 
+    @support_date_range("HOUR_START")
```

Comment:
not sure if it's better for maintainability off the top of my head, but if set `freq=None` you can just use  `published_before` and `published_after` and `_get_documents` to return the set of documents that match and then do concat them. 

if you can get this approach to work, i think it may be more maintainble than using the decorator because there is less magic going on.

Reply:
Updated! Thanks for the suggestion.